news/templates/news/user_news.html

```diff
@@ -2,37 +2,44 @@
 {% load news %}
 
 {% for item in alerts %}
-<div class="alert alert-{{ item.alert }}">
-  <h4>
-    {{ item.title|safe }}
-    {% if not item|is_published:now %}<small><span class="label label-default">{% trans "Appears" %} {{ item.publish }}</span></small>{% endif %}
-    {% if is_course_staff %}<small><span class="label label-default">{{ item.audience|news_audience }}</span></small>{% endif %}
-  </h4>
-  <div>
-    {{ item.body|safe }}
-  </div>
-</div>
+	<div class="alert alert-{{ item.alert }}">
+		<h4>
+			{{ item.title|safe }}
+			{% if not item|is_published:now %}
+				<small><span class="label label-default">{% trans "Appears" %} {{ item.publish }}</span></small>
+			{% endif %}
+			{% if is_course_staff %}
+				<small><span class="label label-default">{{ item.audience|news_audience }}</span></small>
+			{% endif %}
+		</h4>
+		<div>
+			{{ item.body|safe }}
+		</div>
+	</div>
 {% endfor %}
 
 {% if news %}
-<div class="panel panel-default news-panel">
-  <div class="panel-heading">
-    <h4 class="panel-title">{% trans "Course news" %}</h4>
-  </div>
-  <div class="list-group">
-    {% for item in news %}
-    <a class="list-group-item{% if item.collapsed %} collapsed{% endif %}" href="#news{{ item.id }}" data-toggle="collapse" role="button" aria-controls="news{{ item.id }}" aria-expanded="{% if item.collapsed %}false{% else %}true{% endif %}">
-      <h4 class="list-group-item-heading">
-        <span class="caret"></span>
-        {{ item.title|safe }} <small>{{ item.publish }}</small>
-        {% if not item|is_published:now %}<small><span class="label label-default">{% trans "Appears later" %}</span></small>{% endif %}
-        {% if is_course_staff %}<small><span class="label label-default">{{ item.audience|news_audience }}</span></small>{% endif %}
-      </h4>
-    </a>
-    <div class="list-group-item-text collapse{% if not item.collapsed %} in{% endif %}" id="news{{ item.id }}">
-      {{ item.body|safe }}
+	<div class="panel panel-default news-panel">
+    	<div class="panel-heading">
+      		<h3 class="news-panel panel-title">{% trans "Course news" %}</h3>
+    	</div>
+    	<div class="list-group">
+      	{% for item in news %}
+      	<div class="list-group-item">
+        	<h4 class="list-group-item-heading">
+				{{ item.title|safe }} <small>{{ item.publish }}</small>
+				{% if not item|is_published:now %}
+					<small><span class="label label-default">{% trans "Appears later" %}</span></small>
+				{% endif %}
+				{% if is_course_staff %}
+					<small><span class="label label-default">{{ item.audience|news_audience }}</span></small>
+				{% endif %}
+        	</h4>
```

Comment:
This line has spaces.

Reply:
You are correct, that has been fixed. Those two indendation-commits can be squashed into one once this is mergeable.